aioseop_class.php

```diff
@@ -1462,7 +1465,13 @@ function get_aioseop_title( $post, $use_original_title_format = true ) {
 		} elseif ( is_search() && isset( $s ) && ! empty( $s ) ) {
 			$search = esc_attr( stripslashes( $s ) );
 			$title_format = $aioseop_options['aiosp_search_title_format'];
-			$title        = str_replace( '%blog_title%', $this->internationalize( get_bloginfo( 'name' ) ), $title_format );
+			$title        = str_replace( '%site_title%', $this->internationalize( get_bloginfo( 'name' ) ), $title_format );
+			if ( strpos( $title, '%blog_title%' ) !== false ) {
```

Comment:
The parent class method for strpos should be used instead. (multiple occurances)

Using Yoda operations is also a good habit to get into. `if ( 'value === $variable ) {`. It's a WP Standard and values are generally shorter than variable & function names. (multiple occurances)

Reply:
Done.